playbook.yml

```diff
@@ -79,9 +83,47 @@
 
   roles:
     - role: ANXS.postgresql
+      postgresql_version: 9.5
+      postgresql_default_auth_method: md5
+      postgresql_listen_addresses: '*'
+      postgresql_pg_hba_default:
+        - type: local
+          database: all
+          user: "{{ postgresql_admin_user }}"
+          address: ''
+          method: peer
+        - type: local
+          database: all
+          user: all
+          address: ''
+          method: peer
+        - type: host
+          database: all
+          user: all
+          address: 127.0.0.1/32
+          method: "{{ postgresql_default_auth_method }}"
+        - type: host
+          database: all
+          user: all
+          address: ::1/128
+          method: "{{ postgresql_default_auth_method }}"
+        - type: host
+          database: all
+          user: all
+          address: 0.0.0.0/0
+          method: "{{ postgresql_default_auth_method }}"
+        - type: host
+          database: all
+          user: all
+          address: ::0/0
+          method: "{{ postgresql_default_auth_method }}"
```

Comment:
When `ufw` is changed to only allow in-network traffic, these public-facing postgres auth options should go away (for consistency).